spatialos-sdk/src/worker/entity.rs

```diff
@@ -153,7 +159,7 @@ impl RawEntity {
 
         RawEntity {
             components: new_data,
-            database,
+            database: &component::DATABASE,
```

Comment:
Now that `ComponentDatabase` is a singleton (again) it shouldn't be necessary to store references to it in a struct.